platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -46,7 +46,7 @@ android {
     // When building from Android Studio, gradle.properties sets `android.buildOnlyTargetAbi=true` so that
     // only the architecture for the device you're running on gets built.
     def abi = 'all'
-    if (!project.hasProperty('android.injected.invoked.from.ide') || project.hasProperty("mapbox.abis")) {
+    if (project.hasProperty("mapbox.abis")) {
         // Errors when the user invokes Gradle from the command line and didn't set mapbox.abis
```

Comment:
This comment is no longer true then. I think this was put in to force selecting the ABI when building command line. Not sure if that is helpful.

Reply:
Oh, the comment is saying that it's _intended_ to force you to provide `-Pmapbox.abis`? I assumed it was just an odd way to describe the behavior if you don't guard with `hasProperty`.

Is it really useful to force `-Pmapbox.abis`? Seems like an unnecessary stumbling block.

Reply:
The purpose of this is to prevent calling without `-Pmapbox.abis` since it will cause extremely long build times since all architectures are going to be built.

Reply:
🤷‍♂️  OK.